crates/contracts/src/precompiles/tip20.rs

```diff
@@ -98,6 +98,7 @@ crate::sol! {
         function optedInSupply() external view returns (uint128);
         function globalRewardPerToken() external view returns (uint256);
         function userRewardInfo(address account) external view returns (UserRewardInfo memory);
+        function getPendingRewards(address account) external view returns (uint256);
```

Comment:
nit: since total supply is capped at u128 should this return uint128 instead of uint256?

Reply:
sure done